src/object.rs

```diff
@@ -1118,14 +1117,22 @@ impl<T: IsA<Object> + SetValue> ObjectExt for T {
 
             let mut v_args: Vec<Value>;
             let mut s_args: [Value; 10] = mem::zeroed();
+            let self_v = {
+                let mut v = Value::uninitialized();
+                gobject_ffi::g_value_init(v.to_glib_none_mut().0, self.get_type().to_glib());
+                gobject_ffi::g_value_set_object(v.to_glib_none_mut().0, self.to_glib_none().0);
+                v
+            };
             let args = if args.len() < 10 {
-                for (i, arg) in iter::once(&(self as &ToValue)).chain(args).enumerate() {
-                    s_args[i] = arg.to_value();
+                s_args[0] = self_v;
+                for (i, arg) in args.iter().enumerate() {
+                    s_args[i+1] = arg.to_value();
```

Comment:
Shouldn't calling `.all()` instead of looping on it be more rusty?

Reply:
I think it'd work as follows:

```rust
args.iter().enumerate().for_each(|(i, arg)| { s_args[i+1] = arg.to_value(); });
```

(Yes, I know, I used `for_each` instead haha)

Reply:
`for_each` is not really idiomatic

Reply:
It's just nitpicking. Either way is fine. Why isn't it idiomatic btw?

Reply:
For imperative code (which `for_each` is), `for` loops are generally preferred

Reply:
Ok, good to know! Then it's all good for me. I'll merge once CIs passed.